.github/workflows/style_checks.yaml

```
@@ -34,11 +34,11 @@ jobs:
 
       - name: Install packages
         run: |
-          python -m pip install blackdoc docformatter ruff
+          python -m pip install docformatter ruff
           python -m pip list
           sudo apt-get install dos2unix
 
-      - name: Formatting check (blackdoc, docformatter, ruff)
+      - name: Formatting check
```

Maybe keep the `(docformatter, ruff)` part for now, I find it helpful to know what is being run in the CI.
```suggestion
      - name: Formatting check (docformatter, ruff)
```